packages/fxa-auth-server/docs/swagger/oauth-api.ts

```diff
@@ -36,16 +36,19 @@ const OAUTH_DESTROY_POST = {
   plugins: {
     'hapi-swagger': {
       responses: {
+        200: {
+          description: 'No information is returned in the response body.'
+        },
         401: {
           description: dedent`
             Failing requests may be caused by the following errors (this is not an exhaustive list):
-            \`errno: 171\` - Incorrect client secret.
+            - \`errno: 171\` - Incorrect client secret.
```

Comment:
I noticed we had a few of these items that have trailing periods but not all.

And because it done made me suspicious, here's a fun (j/k, it's atrocious) grep that will fetch all unique `errno` strings and their respective codes.

```sh
git grep -Eh "\\\\\`errno:\s[[:digit:]]+\\\\\`" packages | grep "\- \\\\\`errno:" | sort | uniq
```

<details>
<summary>clicko expando 🪄 </summary>

            - \`errno: 101\` - Account already exists
            - \`errno: 102\` - Unknown account
            - \`errno: 103\` - Incorrect password
            - \`errno: 104\` - Unverified account
            - \`errno: 105\` - Invalid verification code
            - \`errno: 105\` - Invalid verification code'
            - \`errno: 107\` - Invalid parameter in request body
            - \`errno: 108\` - Missing parameter in request body
            - \`errno: 110\` - Invalid authentication token in request signature
            - \`errno: 110\` - Invalid authentication token in request signature.
            - \`errno: 116\` - This endpoint is no longer supported
            - \`errno: 125\` - The request was blocked for security reasons
            - \`errno: 127\` - Invalid unblock code
            - \`errno: 138\` - Unverified session
            - \`errno: 139\` - Can not add secondary email that is same as your primary
            - \`errno: 140\` - Email already exists
            - \`errno: 141\` - Email already exists
            - \`errno: 142\` - Sign in with this email type is not currently supported
            - \`errno: 144\` - Email already exists
            - \`errno: 145\` - Reset password with this email type is not currently supported
            - \`errno: 147\` - Can not change primary email to an unverified email
            - \`errno: 148\` - Can not change primary email to an email that does not belong to this account
            - \`errno: 149\` - This email can not currently be used to login
            - \`errno: 150\` - Can not resend code for email that does not belong to the account
            - \`errno: 150\` - Can not resend email code to an email that does not belong to this account
            - \`errno: 151\` - Failed to send email
            - \`errno: 157\` - Unavailable device command
            - \`errno: 160\` - This request requires two step authentication enabled on your account.
            - \`errno: 160\` - This request requires two-step authentication enabled on your account
            - \`errno: 162\` - Unknown client id.
            - \`errno: 171\` - Incorrect client secret.
            - \`errno: 202\` - Feature not enabled
            - \`errno: 998\` - An internal validation check failed.
</summary>

Note that:
- 105 is in there twice (once w/ trailing `'`, once without).
- 110 (and 160) is in there twice (once w/ period, once without).
- 140, 141, and 144 have the same message with different codes.
- 150 is listed twice, with different (yet similar) messages.

Reply:
Hmm, the original documentation has two different messages for 150 - I'll update them so it only has one and also remove any trailing punctuation. Confirming that 140, 141, and 144 all have the same messages in the original doc.

https://github.com/mozilla/fxa/blob/e9e80718641592aef71bb9368031cc4cf913f1c8/packages/fxa-auth-server/docs/api.md